website/docs/components/form/select/index.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: Form::Select
+description: A form element that allows users to choose one option from a list.
+caption: A form element that allows users to choose one option from a list.
```

Comment:
Do we disallow the `multiple` attribute? https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/multiple